package.json

```diff
@@ -80,7 +80,7 @@
     "graphql-schema-typescript": "^1.5.2",
     "graphql-upload": "^12.0.0",
     "helmet": "^4.6.0",
-    "ipfs-http-client": "^50.1.2",
+    "ipfs-http-client": "^56.0.3",
```

Comment:
not the latest `^57.xx.yy` which is ESM only module, seems not compatible with current typescript transpling to target commonjs